lib/ffi-icu/locale.rb

```diff
@@ -183,13 +183,15 @@ def script
       end
     end
 
+    # rubocop:disable Style/OptionalBooleanParameter
     def to_language_tag(strict = false)
       Lib::Util.read_string_buffer(64) do |buffer, status|
         Lib.uloc_toLanguageTag(@id, buffer, buffer.size, strict ? 1 : 0, status)
       end
     end
+    # rubocop:enable Style/OptionalBooleanParameter
```

Comment:
These enable/disable cop comments it can also be done on [one line](https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code).  
```suggestion
    def to_language_tag(strict = false) # rubocop:disable Style/OptionalBooleanParameter
      Lib::Util.read_string_buffer(64) do |buffer, status|
        Lib.uloc_toLanguageTag(@id, buffer, buffer.size, strict ? 1 : 0, status)
      end
    end
```

Reply:
Same with some other files.

Reply:
Thanks!